feconf.py

```diff
@@ -664,7 +664,7 @@ def get_empty_ratings():
 
 # Possible values for user query status.
 # Valid status transitions are: processing --> completed --> archived
-# Or processing --> failed.
+# or processing --> failed.
```

Comment:
@bansalnitish: Just wanted to confirm, this has been done manually right?

Reply:
Yes @apb7, this has been done manually!

Reply:
Thank you, @bansalnitish :)